activities/zio-flow-twilio/shared/src/main/scala/zio/flow/activities/twilio/Message.scala

```diff
@@ -0,0 +1,54 @@
+package zio.flow.activities.twilio
+
+import zio.flow._
+import zio.schema.DeriveSchema
+
+final case class Message(
+  body: String,
+  num_segments: String,
+  direction: String, // TODO: should be Direction serialized into a single string field
+  from: PhoneNumber,
+  to: PhoneNumber,
+  date_updated: String, // TODO: should be Instant and parsed using DateTimeFormatter.RFC_1123_DATE_TIME
```

Comment:
To be able to do anything with these fields, it should be `Instant` because that's what we currently have remote syntax for. But we can't do that without specifying the date-time formatter for the Json codec, and we cannot do that here.

Maybe with a custom wrapper with a custom schema it can be done - but that introduces one more layer to unpack with an accessor before being able to work with the instant. To be explored.

Reply:
I never liked having to specify date/time formatter for `java.time` entities (it almost biases toward string serialization). Maybe we can reconsider that?